### PR TITLE
Adding glome key version and console state.

### PR DESCRIPTION
**Pull request by sourcequench**
### Change Scope

* Adding key and glome-console state to telemetry, for use in an upcoming gnsi.credentialz change
* This is backwards compatible

There are no implementations yet.

### CODE REVIEW

**Comment by dplore** [comment]
@morrowc could you give a review?

**Review by dplore — COMMENTED** [review]
Hi, need to bump the version and reference here: https://github.com/openconfig/public/pull/1178/files#diff-6542718fbe869ecef8de06948a5973b627fdf3ff7de2f047bc7ccd4349c9e940R42-R50

**Comment by sourcequench** [comment]
Done.

**Review by morrowc — APPROVED** [review]

**Comment by morrowc** [comment]
"This branch is out-of-date with the base branch" - I suspect a git sync/merge is necessary before submit.

**Comment by sourcequench** [comment]
Done. Branch is in sync, save for my changes.

File: release/models/gnsi/openconfig-gnsi-credentialz.yang [hunk 1]
```diff
@@ -232,9 +258,16 @@ module openconfig-gnsi-credentialz {
           "Console-related state.";
 
         uses counters;
+
+        leaf glome {
+          type boolean;
+          description
+            "Whether GLOME is configured or not.";
```
Comment (dplore) [inline release/models/gnsi/openconfig-gnsi-credentialz.yang:265]: Can you define what configured means?  Do you mean if glome is enabled?  If yes, in OC it's most common to have an 'enabled' leaf.

Comment (sourcequench) [reply release/models/gnsi/openconfig-gnsi-credentialz.yang]: Done, changed to enabled.

File: release/models/gnsi/openconfig-gnsi-credentialz.yang [hunk 2]
```diff
@@ -41,6 +41,12 @@ module openconfig-gnsi-credentialz {
 
   oc-ext:openconfig-version "0.6.0";
```
Comment (dplore) [inline release/models/gnsi/openconfig-gnsi-credentialz.yang:42]:
```suggestion
  oc-ext:openconfig-version "0.7.0";
```

Comment (dplore) [reply release/models/gnsi/openconfig-gnsi-credentialz.yang:42]: This will fix the CI check and then I think this is ready.

**Review by dplore — APPROVED** [review]

**Pull request merged**